tutorial/index.html

```diff
@@ -0,0 +1,412 @@
+<html>
+<head>
+    <title>Akka HTTP to Persistent Actor to Akka Stream to Websocket</title>
+</head>
+<body>
+<div>
+    <h2>Getting started with this template</h2>
+
+    <p>
+        In this template you will learn some basics of
+        <strong>
+            <ul>
+                <li>Akka Http</li>
+                <li>Akka Persistence</li>
+                <li>Reactive Streams</li>
+                <li>Websockets</li>
+            </ul>
+        </strong>
+    </p>
+    <p>
+        This template consists of two parts, a theoretical and a hands-on part.
+        In the theoretical part you will learn about the techniques mentioned above by means of a fully working example.
+        In the hands-on part you will test-drive your own flow.
+        So don't mind the failing <a href="#test">tests</a> for now, you will fix them soon enough ;)
+    </p>
+    <p>
+        Next: <a href="#tutorial/1" class="shortcut">An Overview</a>
+    </p>
+</div>
+
+<div>
+    <h2>Part 1: An Overview</h2>
+
+    <p>
+        In this tutorial we will use a simple Twitter like application to demonstrate the different techniques. You can
+        post a tweet as a user and monitor a reactive timeline that changes whenever a new tweet is posted. We
+        simplified
+        the functionality a bit to keep the focus on the essentials.
+    </p>
+
+    <h3>Posting tweets</h3>
+
+    <p>
+        You can post tweets via a simple REST endpoint. A tweet consists of a username and the message. We use
+        <strong>Akka Persistence</strong> to keep track of the tweets for each user and be able to recover the
+        timeline for a user after a restart of the application. All incoming tweets are put on the event bus that comes
+        with <strong>Akka</strong> once they are persisted.
```

Comment:
So if I understand correctly: *first* they're persisted, *then* they're put on the event bus?